### DISCUSSION TITLE
doc site look to be down

### DISCUSSION THREAD

**Discussion by dbones** [dotnet/dotnet-operator-sdk#509] [Ideas & General Discussion]
https://buehler.github.io/dotnet-operator-sdk/

![image](https://user-images.githubusercontent.com/183849/213465690-f52257c8-7544-4920-bdfc-cacecea61d26.png)

(not sure if you have moved it, or if you want this raised as a ticket)

**Comment by buehler** [comment]
Thank you, I moved it into the README files of all packages. Then I unpublished the old site.
